### PR TITLE
Feature/shield

### PR DESCRIPTION
**Pull request by szkript**

### CODE REVIEW

**Comment by sourabhv** [comment]
Can you add some description to your PR? If this is a new gamemode, please add it as a link to your fork in README

File: flappy.py [hunk 1]
```diff
@@ -89,11 +94,11 @@ def main():
     else:
         soundExt = '.ogg'
 
-    SOUNDS['die']    = pygame.mixer.Sound('assets/audio/die' + soundExt)
-    SOUNDS['hit']    = pygame.mixer.Sound('assets/audio/hit' + soundExt)
-    SOUNDS['point']  = pygame.mixer.Sound('assets/audio/point' + soundExt)
```
Comment (sourabhv) [inline flappy.py:94]: Can you undo these unneeded changes?

File: flappy.py [hunk 2]
```diff
@@ -214,22 +219,29 @@ def mainGame(movementInfo):
...
     # player velocity, max velocity, downward accleration, accleration on flap
-    playerVelY    =  -9   # player's velocity along Y, default same as playerFlapped
-    playerMaxVelY =  10   # max vel along Y, max descend speed
-    playerMinVelY =  -8   # min vel along Y, max ascend speed
-    playerAccY    =   1   # players downward accleration
-    playerRot     =  45   # player's rotation
-    playerVelRot  =   3   # angular speed
-    playerRotThr  =  20   # rotation threshold
-    playerFlapAcc =  -9   # players speed on flapping
-    playerFlapped = False # True when player flaps
```
Comment (sourabhv) [inline flappy.py:225]: These too

File: flappy.py [hunk 3]
```diff
@@ -76,6 +78,9 @@ def main():
         pygame.image.load('assets/sprites/9.png').convert_alpha()
     )
 
+    # load shield effect sprite
+    IMAGES['shield'] = pygame.image.load(
+        'C:/Users/szkri/PycharmProjects/FlapPyBird/assets/sprites/shield.png').convert_alpha()
```
Comment (sourabhv) [inline flappy.py:83]: This is an absolute path on your system, add a path from assets, like above 

```python
BACKGROUNDS_LIST = (
    'assets/sprites/background-day.png',
    'assets/sprites/background-night.png',
)
```

**Comment by szkript** [comment]
sorry , I don't wanted to mess around, i tought it was uploading to the forked repo,I'm not too experienced with it, just wanted to expand the inputs of the game for experimenting purposes:D
I'll try to fix it

**Comment by sourabhv** [comment]
It is uploading to a forked repo. But you have an open PR, thats why I got an update for this. If you don't intend to send a PR, you can close it

**Comment by szkript** [comment]
I think it's closed, sorry for bothering you

Sourabh Verma <notifications@github.com> ezt írta (időpont: 2020. jún. 12.,
P, 13:27):

**Pull request closed without merging**